include/mbedtls/ssl.h

```diff
@@ -124,6 +126,10 @@
 
 #define MBEDTLS_SSL_MAX_HOST_NAME_LEN           255 /*!< Maximum host name defined in RFC 1035 */
 
+#define MBEDTLS_SSL_MAX_SIG_HASH_ALGORITHMS     65534 /*!< Maximum size of list in sig-hash algorithm ext., in RFC 5246    */
+#define MBEDTLS_SSL_MAX_SUPPORTED_ECC           65534 /*!< Maximum size of list in supported elliptic curve ext., RFC 4492 */
+#define MBEDTLS_SSL_MAX_ALPN                    65534 /*!< Maximum size of list in alpn ext., RFC 7301                     */
```

Comment:
Like `MBEDTLS_SSL_MAX_HOST_NAME_LEN` these are constants from the standard, so I think it's ok to add them here.